src/main/resources/META-INF/plugin.xml

```diff
@@ -11,6 +11,14 @@
   ]]></description>
 
   <change-notes><![CDATA[
+    <p><b>1.20.0</b></p>
+    <ul>
+      <li>Support for Renarde Quarkus application (URL codelens in Java file, uri/uriabs support in Qute template)</li>
+      <li>Support for Qute fragments</li>
+      <li>Support Quarkus3 for Qute</li>
+      <li>Completion context aware for snippets</li>
```

Comment:
snippets are not rendered correctly so I'm not sure it's worth mentioning

Reply:
removed